kernel/core/utils/stringtools.hpp

```diff
@@ -213,6 +227,12 @@ std::string ellipsize(const std::string& sLongString, size_t nMaxStringLength =
 
 bool isEqualStripped(StringView str1, StringView str2);
 
+// NEW
+extern int last_num_format;
+extern int num_format_votes[];
+static void voteNumType(int numType);

```

Comment:
You do not declare `static` functions in a header (with rare exceptions*). The purpose of `static` functions is to make them local-only. This has the benefit of having lower interdependencies as well as the compiler is able to optimize better (bc. it knows all possible usages of this function).

(* There are cases, when `static` functions are written in a header file, that is: this header file is an actual implementation file and only used in a single other implementation file.)